jbpm-wb-kie-server/jbpm-wb-kie-server-backend/src/test/java/org/jbpm/workbench/ks/integration/KieServerQueryDefinitionLoaderTest.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jbpm.workbench.ks.integration;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.enterprise.event.Event;
+
+import org.jbpm.workbench.ks.integration.event.QueryDefinitionLoaded;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.kie.server.api.model.definition.QueryDefinition;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doAnswer;
+import static org.mockito.MockitoAnnotations.openMocks;
+
+@RunWith(MockitoJUnitRunner.class)
+public class KieServerQueryDefinitionLoaderTest {
+
+    private final Map<String, QueryDefinition> receivedEvents = new HashMap<>();
+
+    @Mock
+    private Event<QueryDefinitionLoaded> event;
+
+    @InjectMocks
+    private KieServerQueryDefinitionLoader kieServerQueryDefinitionLoader;
+
+    @Before
+    public void setup() {
+        openMocks(this);
+    }
+
+    @Test
+    public void testJbpmProcessInstancesWithVariables() {
```

Comment:
Please add such test for all the named queries similarly to what you did for `jbpmProcessInstancesWithVariables` here

Reply:
done